postmaster/forms.py

```diff
@@ -16,6 +16,7 @@ class LoginForm(Form):
     """
     username = StringField(label='Username', validators=[DataRequired()])
     password = PasswordField(label='Password', validators=[DataRequired(), validate_wtforms_password])
+    two_factor = IntegerField(label='2 Factor', validators=(validators.Optional(),))
```

Comment:
I'm not in love with this always being shown on the login screen as an input field. I would prefer if this was a modal that appeared after a successful password was entered.

We can discuss this further.